test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -383,7 +383,7 @@ public void ItCreatesCoverageFileInResultsDirectory()
             result.ExitCode.Should().Be(1);
         }
 
-        [UnixOnlyFact(Skip = "https://github.com/dotnet/cli/issues/9773")]
```

Comment:
Remember to close https://github.com/dotnet/cli/issues/9773 when this PR goes in.

Reply:
Done.